src/VisualStudio/TestUtilities/VisualStudioInstance.cs

```diff
@@ -136,6 +149,7 @@ public void Close()
 
         public void CloseAndDeleteOpenSolution()
         {
+            WaitForApplicationIdle();
```

Comment:
Why? If there's *every* a wait there should always be a clear explanation as to why it's necessary in a comment. Random waits like this is the beginning of the end of any integration test framework. (I'd like that not to start now. :-))